airbyte-integrations/connectors/source-okta/source_okta/source.py

```diff
@@ -180,9 +184,11 @@ def request_params(
         # The log stream use a different params to get data
         # https://developer.okta.com/docs/reference/api/system-log/#datetime-filter
         stream_state = stream_state or {}
-        params = OktaStream.request_params(self, stream_state, stream_slice, next_page_token)
+        params = super().request_params(stream_state, stream_slice, next_page_token)
         latest_entry = stream_state.get(self.cursor_field)
-        if latest_entry:
+        if self.default_start_date and not latest_entry:
```

Comment:
is this only available on the `Logs` stream and not in `IncrementalOktaStream`? it is a bit misleading to have the `start_date` input for the whole connector but only be applied for this one stream.

Reply:
@drrest reminder about this question

Reply:
@sherifnada Updated.